src/content/en/updates/2020/03/devtools.md

```diff
@@ -0,0 +1,153 @@
+they went to try it out they couldn't find the "Lighthouse" panel, so the **Audits** panel is now the
+**Lighthouse** panel.
+
+![The Lighthouse panel](/web/updates/images/2020/03/lighthouse.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/2ae53ef04d84c4201836eb63238e97bc180ea54d #}
+
+<!--
+
+### New audits {: #new-audits }
+
+New audits:
+
+* [CLS](https://github.com/GoogleChrome/lighthouse/commit/a8a0f8fd369faaa23b97d8803bfc220db31e4769#diff-eb7e207cabf22c8000b5abe8f18c6daf)
+* [Legacy JS](https://github.com/GoogleChrome/lighthouse/pull/10303)
+* [Maskable icon](https://github.com/GoogleChrome/lighthouse/pull/10370)
+* [Duplicate JS](https://github.com/GoogleChrome/lighthouse/commit/ea9d226606c5b534d4259253baac5f709a9e3d7c#diff-eb7e207cabf22c8000b5abe8f18c6daf)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/933525c2bdcf22d49b5688c04cad7bc30c476b3d #}
+
+Send feedback to [Chromium issue #772558](https://crbug.com/772558).
+
+-->
+
+## Delete all Local Overrides in a folder {: #overrides }
+
+After setting up [Local Overrides](/web/updates/2018/01/devtools#overrides) you can now
+right-click a folder and select the new **Delete all overrides** option to delete all
+Local Overrides in that folder.
+
+![Delete all overrides](/web/updates/images/2020/03/overrides.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/0fc949c8b168dea42d12432f4591ee4c5dabb9a5 #}
+
+Send feedback to [Chromium issue #1016501](https://crbug.com/1016501).
+
+## Updated Long tasks UI {: #long-tasks }
+
+A [Long Task](https://web.dev/long-tasks-devtools/#what-are-long-tasks) is JavaScript code that
+monopolizes the main thread for a long time, causing a web page to freeze.
+
+You've been able to [visualize Long Tasks in the Performance panel](https://web.dev/long-tasks-devtools/#are-there-long-tasks-in-my-page-that-could-delay-interactivity)
+for a while now, but in Chrome 82 the Long Task visualization UI in the Performance panel has been updated.
+The Long Task portion of a task is now colored with a striped red background.
+
+![The new Long Task UI](/web/updates/images/2020/03/long-task.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e19afdbb2fb24f4123dc7b8b62f38d359ff2d916 #}
+
+Send feedback to [Chromium issue #1054447](https://crbug.com/1054447).
+
+## Maskable icon support in the Manifest pane {: #maskable-icons }
+
+Android Oreo introduced adaptive icons, which display app icons in a variety
+of shapes across different device models. [Maskable icons](https://web.dev/maskable-icon/)
+are a new icon format that support adaptive icons, which will enable you to ensure that
+your [PWA](https://web.dev/progressive-web-apps) icon looks good on Android Oreo devices.
+
+Enable the new **Show only the minimum safe area for maskable icons** checkbox in the
+**Manifest** pane to check that your maskable icon will look good on Android Oreo
```

Comment:
maybe “on devices that support the maskable icons standard, such as Android Oreo devices”

Reply:
Done